commit 67f604e3a0c69f6d3acd723175c76cf003c1e66f (refs/changes/14/5714/2)
Author: Jerry Jelinek <jerry.jelinek@joyent.com>
Date:   2019-03-05T16:56:11+00:00 (7 months ago)
    
    OS-7634 enable zfs-test cliroot zpool_get_002_pos
    Reviewed by: Kody Kantor <kody.kantor@joyent.com>
    Approved by: Kody Kantor <kody.kantor@joyent.com>

diff --git a/usr/src/test/zfs-tests/runfiles/smartos.run b/usr/src/test/zfs-tests/runfiles/smartos.run
index 2443f6fc6a..18a491c403 100644
--- a/usr/src/test/zfs-tests/runfiles/smartos.run
+++ b/usr/src/test/zfs-tests/runfiles/smartos.run
@@ -24,7 +24,7 @@ post = cleanup
 outputdir = /var/tmp/test_results
 
 [/opt/zfs-tests/tests/functional/cli_root/zpool_get]
-tests = ['zpool_get_001_pos', 'zpool_get_003_pos',
+tests = ['zpool_get_001_pos', 'zpool_get_002_pos', 'zpool_get_003_pos',
     'zpool_get_004_neg']
 
 [/opt/zfs-tests/tests/functional/cli_root/zpool_import]
